README.md

```diff
@@ -42,6 +42,10 @@ A custom show can be run on a supported vehicle by loading it via a USB flash dr
         - Boolean light
         - Ramping light
         - Closures
+        
+## Audio file requirements
+You can use both the mp3 and wav format (.wav is recommended)
+Make sure the file is encoded with a sample rate of 44.1Mhz; less common 48Mhz files won't properly sync to the light show.

```

Comment:
Should be kHz not Mhz.
And add a space between the number and unit (e.g., 44.1 kHz) and we can merge this documentation update.